src/components/avatar/index.js

```diff
@@ -39,6 +50,14 @@ export default class Avatar extends BaseComponent {
      * Background color for Avatar
      */
     backgroundColor: PropTypes.string,
+    /**
+     * Badge location on Avatar
+     */
+    badgePosition: PropTypes.oneOf(Object.keys(BADGE_POSITIONS)),
```

Comment:
although they are the same, it should be `Object.values` instead of keys. 
since what's being passed eventually is the value 
Theoretically, if someone will change the values, your code will be broken